plugins/chat/assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -77,15 +78,8 @@ export default class ChatComposer extends Component {
   }
 
   @action
-  sendMessage(raw) {
-    const message = ChatMessage.createDraftMessage(this.args.channel, {
-      user: this.currentUser,
-      message: raw,
-    });
-
-    this.args.onSendMessage(message);
-
-    return Promise.resolve();
+  sendMessage() {
+    this.args.onSendMessage(this.currentMessage);
```

Comment:
We don't need to create a fake message here, we can't just keep using the draft we have